### ISSUE TITLE
Task Schedule

### ISSUE THREAD

**Issue by gastondg** [basildane/WakeOnLAN#94]
I tried to schedule a task for "Start Computer" with this options:
Name: "start"
Description: "prender equipo"
Security Options: login user + password.
Trigger:
--One time at 2/26/18 11:00:00 AM 
Action:
--Start Computer --> Details: "equipo de testing"

I allways get the same error massage:
"El servicio de Programador de tareas intentó ejecutar la tarea, pero no pudo hacerlo debido a una restricción en la definición de la tarea. (0x80041324)"
Translation: "Schedule service couldn't execute the task because there is a restriction on the task definition"

I don't even know where you can put a restriction for the task!

Help please!

WOL Version: 2.11.20

**Comment by basildane** [comment]
Yes, scheduled tasks run under account permissions (usually).  When you create tasks with WOL, you are asked to enter a username and password that will be used for the scheduled tasks.

Just for testing, could you try using an Administrator level account and see if that fixes it?

**Comment by gastondg** [comment]
I'm using an Administrator level account now and the proccess ends with a "Task executed correctly", but in the main window of the program, you can see a message with a label saying: "Error in remote server (407): Proxy authentication required".
I have the proxy configured in windows 7, but I don't know if the WOL doesn't support it.
Any advice?

**Comment by gastondg** [comment]
Also, this is only for Shutdown.

**Comment by basildane** [comment]
To simplify testing, let's leave the task scheduler out of it for now.  Just try to shutdown from the main program.

There are different shutdown methods supported.  WMI, Legacy.  Which one are you using?

**Comment by basildane** [comment]
"Proxy authentication required" is an error that should only occur with internet or http requests.  WOL is returning this error?

**Comment by gastondg** [comment]
I can shutdown correctly one by one in the main program, using WMI method. Legacy is supported too but I don't use it.

The proxy authentication error also appears when it tries to update the program. 

Can you give me the update url? 

Thanks

**Comment by gastondg** [comment]
Now I'm having a "task executed correctly" but it doesn't shutdown the computer. 
It's really weird that using main program shutdown works but schedule task does not. 
Any ideas?

**Comment by basildane** [comment]
Update check url is: https://wol.aquilatech.com/updates/AppCast.xml

**Comment by gastondg** [comment]
Thanks a lot.

I resolve the problem changing method. I now use Legacy method.

But it's weird that using WMI method in main program I don't have any problems, but using it in Schedule Task, I do.

**Comment by basildane** [comment]
I don't know.  Have you tried exiting the WOL program to ensure that it wrote the changes to the database, then try the scheduled task with WMI?

I've not had this problem yet.

**Comment by gastondg** [comment]
I tried, nothing happend.
But it's working with Legacy method, so it's okay.

**Comment by basildane** [comment]
OK, I'll look at the code when I have some time, but thanks for letting me know.  Maybe I can test this tonight.
